src/features/moviesPage/moviePage/mainMovieTile/index.js

```diff
@@ -0,0 +1,44 @@
+import { imagesAPI600x900 } from "../../../../core/API";
+import { ButtonTag, MovieDetails, MovieTitle, Overview, Poster, PrimaryInfo, Production, ProductionCountry, Rate, Rating, ReleaseYear, Star, Tag, Tags, Votes, Wrapper } from "./styled";
+import star from "../../../../common/images/star.svg"
+
+const MainMovieTile = ({ poster, title, date, genres, productionCountry, release, rate, voteCount, overview }) => {
+
+    return (
+        <Wrapper>
+            <Poster src={`${imagesAPI600x900}${poster}`} />
+            <MovieDetails>
+                <MovieTitle>{title}</MovieTitle>
+                <ReleaseYear>{date.slice(0, 4)}</ReleaseYear>
+                <PrimaryInfo>
+                    <Production>Production: </Production>
+                    <ProductionCountry>
+                        {productionCountry.map((country) => country.name).join(", ")}
+                    </ProductionCountry>
+                </PrimaryInfo>
+                <PrimaryInfo>
+                    <Production>Release date: </Production>
+                    <ProductionCountry>
+                        {new Date(release).toLocaleDateString()}
+                    </ProductionCountry>
+                </PrimaryInfo>
+                <Tags>
+                    {genres.map((genre) => (
+                        <ButtonTag key={genre.id}>
```

Comment:
here a unique key is needed and this one will be repeated on the website. I suggest using `${genere.id}${movie.id}`